src/olympia/constants/blocklist.py

```diff
@@ -6,3 +11,10 @@
 MLBF_BASE_ID_CONFIG_KEY = 'blocklist_mlbf_base_id'
 
 REMOTE_SETTINGS_COLLECTION_MLBF = 'addons-bloomfilters'
+
+
+# As BlockVersion.BLOCK_TYPE_CHOICES except meant to be exposed to end-users.
+BLOCK_TYPE_END_USERS_CHOICES = Choices(
+    ('BLOCKED', 0, _('Blocked')),
+    ('SOFT_BLOCKED', 1, _('Restricted')),
+)
```

Comment:
I agree. As much as I don't like switching the field to an integer, we could do it, that would solve the most confusing part. But I don't see a way to solve the strings, since UX want us to say "Restricted" to end-users, but internally "Restricted" is such a loaded term...

Reply:
https://github.com/mozilla/addons/issues/15111

Reply:
Re: the label we expose to internal vs end users, we _could_ use the constant name (i.e. SOFT_BLOCKED) but I'm not sure that's a great solution either 🤷